src/main/java/wooteco/subway/admin/service/GraphService.java

```diff
@@ -0,0 +1,58 @@
+package wooteco.subway.admin.service;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+import org.springframework.stereotype.Service;
+
+import wooteco.subway.admin.domain.CriteriaType;
+import wooteco.subway.admin.domain.CustomEdge;
+import wooteco.subway.admin.domain.Line;
+import wooteco.subway.admin.domain.LineStation;
+import wooteco.subway.admin.dto.GraphResultResponse;
+
+@Service
+public class GraphService {
+    public GraphResultResponse findPath(List<Line> lines, Long source, Long target,
+        CriteriaType type) {
+        WeightedMultigraph<Long, CustomEdge> graph = new WeightedMultigraph(
+            DefaultWeightedEdge.class);
```

Comment:
```suggestion
        WeightedMultigraph<Long, CustomEdge> graph = new WeightedMultigraph<>(
                CustomEdge.class);
```

제네릭을 사용하여 타입세이프한 코드를 만들어주세요!